### PR TITLE
Fixes for #64, #62 and #21

### PR DESCRIPTION
**Pull request by ananay**
Added Bower to README

### CODE REVIEW

**Comment by ananay** [comment]
@championswimmer

**Comment by ananay** [comment]
#64

**Comment by ananay** [comment]
Fixes #64 and #62

File: app/pods/components/about-course/component.js [hunk 1]
```diff
@@ -1,9 +1,22 @@
 import Component from '@ember/component';
+import { inject } from '@ember/service';
 import $ from 'jquery';
+import env from "codingblocks-online/config/environment";
 
 export default Component.extend({
   availableRuns: [],
 
+  loginUrl: `${env.oneauthURL}/oauth/authorize?response_type=code&client_id=${env.clientId}&redirect_uri=${env.publicUrl}`,
+  api: inject(),
```
Comment (abhishek97) [inline app/pods/components/about-course/component.js]: No need to inject `api` and `currentUser` services here.

**Comment by ananay** [comment]
@abhishek97 Done!

**Comment by ananay** [comment]
Added fix for https://github.com/coding-blocks/codingblocks.online.projectx/issues/21

**Comment by ananay** [comment]
Separating the pull requests for different issues

**Pull request closed without merging**